### PR TITLE
Update NXDNHosts.txt

### PR DESCRIPTION
**Pull request by W4END**
Adding line 81 and 82 as follows ...

# 805 US RocketCity-AL-NXDN Bridge
805 45.32.166.46 41400

Thanks W4END

### CODE REVIEW

**Comment by W4END** [comment]
Sorry, we had to update it to new IP address

**Comment by W4END** [comment]
Thanks a ton Jonathan.  I have another question if able...on OpenSpot I can talk freely TX and RX with anyone. On pi-star I am unable to RX station.  What setting am I missing?

**Comment by W4END** [comment]
Jonathan, Sorry if this is a repeat....

Thanks a ton Jonathan. I have another question if able...on OpenSpot I can
talk freely TX and RX with anyone. On pi-star I am unable to RX station.
What setting am I missing?

Paul - W4END

**Comment by g4klx** [comment]
Hi Paul
You'll have to take that up on the Pi-Star forums. They make adaptations of my software so they're best places to answer your question.
Jonathan  G4KLX

    On Saturday, 20 January 2024 at 21:07:34 GMT, W4END ***@***.***> wrote:  
 
 

Jonathan, Sorry if this is a repeat....

Thanks a ton Jonathan. I have another question if able...on OpenSpot I can
talk freely TX and RX with anyone. On pi-star I am unable to RX station.
What setting am I missing?

Paul - W4END

**Comment by W4END** [comment]
Ok thanks for the quick reply.  Much appreciated.

I’ll be adding a P25 TG805 to that same IP address in the next week or so.

Again, thanks for the help and all you do.

Paul - W4END

**Comment by W4END** [comment]
Jonathan,  I sent an update to the IP address to be changed for TG805  to this IP address.   45.32.166.46 41400

I think the original IP address was entered instead of this one.

**Comment by W4END** [comment]
Jonathan,  Again, sorry if this is a repeat email...also added to your
site...


Jonathan, I sent an update to the IP address to be changed for TG805 to
this IP address. 45.32.166.46 41400

I think the original IP address was entered instead of this one.


Paul - W4END

File: NXDNGateway/NXDNHosts.txt [hunk 1]
```diff
@@ -78,6 +78,9 @@
 # 730 Chile NXDN Reflector to BM TG 730
 730	sdradio.cl  41400
 
+# 805 US RocketCity-AL-NXDN Bridge
+805 155.138.207.101 41400
```
Comment (W4END) [inline NXDNGateway/NXDNHosts.txt:82]: This should be 805      45.32.166.46     41400

Sorry for the confusion. Had to change IP addresses mid build.

**Comment by g4klx** [comment]
Can you open a new PR for this change please, and don't use already merged ones.

**Comment by W4END** [comment]
It keeps rerouting me to one I have already sent...

Let me try a different path. If it doesn't work, below is how it should
look.

# 805 US RocketCity-AL-NXDN Bridge
805 45.32.166.46 41400

Paul - W4END

**Pull request merged**